pynicotine/gtkgui/ui/settings/userinterface.ui

```diff
@@ -34,7 +34,7 @@
                   <object class="GtkCheckButton" id="DarkMode">
                     <property name="label" translatable="yes">Prefer dark mode</property>
                     <property name="visible">True</property>
-                    <property name="tooltip_text" translatable="yes">Note that the operating system&apos;s theme may take precedence.</property>
+                    <property name="tooltip_text" translatable="yes">Nicotine+ can only ask GTK to enable the dark theme, it cannot force the light theme, and the operating system&apos;s theme may take precedence.</property>
```

Comment:
```suggestion
                    <property name="tooltip_text" translatable="yes">Disabling this option will use the default color scheme of the operating system.</property>
```

Reply:
I don't know if my suggestion is much better, but I'd like to avoid mentioning technical details that result in more questions for the average user, especially outside Linux-based operating systems (what is GTK?).

Reply:
In this case, it is very relevant information to mention "GTK" here, because for example in KDE, there is a dedicated control panel for GTK Appearance settings.